solutions/usaco-570.mdx

```diff
@@ -27,9 +27,10 @@ using namespace std;
 
 int N;
 int litRooms = 1;
-bool visited[101][101];
-bool illuminated[101][101];
-vector<pair<int,int>> switches[101][101];
+const int MN = 101;
```

Comment:
i take it 0-indexed is too hard lol?

Reply:
I thought it would make it easier to explain in the video if it's 1-indexed but I can change it if that's not preferred